tests/subsys/logging/log_core_additional/src/log_test.c

```diff
@@ -34,6 +35,8 @@ struct backend_cb {
 	uint16_t exp_severity[4];
 	/* inform put() to check domain_id of message */
 	bool check_domain_id;
+	/* in asnyc mode, make sure all logs have been handled */
```

Comment:
async

Also this comment should explain what this field is actually for

Reply:
Thank you @sjg20 ! updated.